bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,978 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ferroamp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="ehub-wloadconsq-l1">
```

Comment:
Many types can be combined as only the label differs. The Label can be set on the channel declaration.

For instance: ehub-wloadconsq-l1, ehub-wloadconsq-l2 and ehub-wloadconsq-l3 are all the same except for the label.

Create one combined channel type:
```
	<channel-type id="ehub-wloadconsq">
		<item-type>Number:Energy</item-type>
		<label>WLoadConsq</label>
		<category>Energy</category>
		<state readOnly="true" pattern="%.2f %unit%"/>
	</channel-type>
```

and use it in the thing structure as:

```
    <channel typeId="ehub-wloadconsq" id="ehubwloadconsql1"/>
        <label>WLoadConsq L1</label>
    </channel>
    <channel typeId="ehub-wloadconsq" id="ehubwloadconsql2"/>
        <label>WLoadConsq L2</label>
    </channel>
    <channel typeId="ehub-wloadconsq" id="ehubwloadconsql3"/>
        <label>WLoadConsq L3</label>
    </channel>
```

This would reduce the amount of types dramatically

Reply:
See also https://github.com/openhab/openhab-addons/pull/16258#discussion_r1615144427 regarding system channel types.

Reply:
Hi,
done changes in channels.xml, regarded to reducing types.
I had to do it as in channel.xml otherwise mine IDE was complaining, hope this is OK.

Reply:
Please see commit at 20240607